spring-cloud-task-core/src/main/java/org/springframework/cloud/task/configuration/SimpleTaskConfiguration.java

```diff
@@ -34,12 +39,22 @@
  *
  * @author Glenn Renfro
  */
+@EnableTransactionManagement
 @Configuration
 public class SimpleTaskConfiguration {
 
 	@Autowired
 	private ApplicationContext context;
 
+	@Autowired(required = false)
+	private Collection<DataSource> dataSources;
```

Comment:
Should we provide setter for the `DataSource` so users can provide their own? E.g. you have more than one, but you can provide the *right* one, so you can still use the class.